packages/cli/test/integration/generators/copyright.integration.js

```diff
@@ -202,28 +202,36 @@ restricted by GSA ADP Schedule Contract with <%= owner %>.
       '"copyright.owner": "ACME Inc."',
     );
 
-    /*
-    Copyright (c) ACME Inc. 2020. All Rights Reserved.
-    Node module: myapp
-    This project is licensed under the ISC License, full text below.
-    */
-    assert.fileContent(
-      path.join(sandbox.path, 'LICENSE'),
-      'This project is licensed under the ISC License, full text below.',
-    );
-    assert.fileContent(
-      path.join(sandbox.path, 'LICENSE'),
-      `Copyright (c) ACME Inc. ${year}.`,
-    );
-    assert.fileContent(
+    const licenseText = fs.readFileSync(
       path.join(sandbox.path, 'LICENSE'),
-      'Node module: myapp',
+      'utf-8',
     );
-    assert.fileContent(
-      path.join(sandbox.path, 'LICENSE'),
-      `Permission to use, copy, modify, and /or distribute this software for any
+    expect(licenseText).to.equal(
+      `
+Copyright (c) ACME Inc. ${year}.
+Node module: myapp
+This project is licensed under the ISC License, full text below.
+
+--------
+
+ISC License
+
+ISC License Copyright (c) 2004-2010 by Internet Systems Consortium, Inc. ("ISC")
+
+Copyright (c) 1995-2003 by Internet Software Consortium
```

Comment:
On this line, I think the copyright holder should be ACME Inc., not Internet Software Consortium. Fixing that is out of scope of this pull request though.

Reply:
See https://opensource.org/licenses/ISC